lib/nebulex/caching.ex

```diff
@@ -20,6 +20,11 @@ defmodule Nebulex.Caching do
     * `:opts` - Defines the cache options that will be passed as argument
       to the invoked cache function (optional).
 
+  * `:match` - Defines a function that takes one argument and will be used to decide if the
```

Comment:
I think you forgot to indent here

Reply:
oops! too bad `mix format` does not catch those :). Fixed it